### PR TITLE
OSSM 3.0 TP1: OSSM-8039 [DOC] Additional Prereq needed for Configuring OpenShift Monitoring with Kiali

### PR DESCRIPTION
**Pull request by gwynnemonahan**
**OSSM 3.0 TP1**

[OSSM-8039](https://issues.redhat.com//browse/OSSM-8039) [DOC] Additional Prereq needed for Configuring OpenShift Monitoring with Kiali

This PR is part of the standalone doc set for the OpenShift Service Mesh project. Kathryn is aware that this content applies for a product that is part of a Technology Preview release. The project is seeking feedback from early adopters.

**Merge PR to**: https://github.com/openshift/openshift-docs/tree/service-mesh-docs-main

**And cherry picked to**: https://github.com/openshift/openshift-docs/tree/service-mesh-docs-3.0.0tp1

Version(s):

Technology Preview

**Service Mesh 3.0 is moving to the stand alone format and will not be cherry-picked back to OCP core branches.**

Issue:
https://issues.redhat.com/browse/OSSM-8039

Link to docs preview:
https://81255--ocpdocs-pr.netlify.app/openshift-service-mesh/latest/kiali/ossm-kiali-assembly.html#ossm-config-openshift-monitoring-kiali_ossm-kiali-assembly 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ossm-config-openshift-monitoring-kiali.adoc [hunk 1]
```diff
@@ -20,6 +20,8 @@ The following steps show how to integrate the {KialiProduct} with user-workload
 
 * User-workload monitoring is enabled. See link:https://docs.openshift.com/container-platform/4.16/observability/monitoring/enabling-monitoring-for-user-defined-projects.html[Enabling monitoring for user-defined projects].
 
+* OpenShift Monitoring has been configured with {SMProductShortName}.
```
Comment (nrfox) [inline modules/ossm-config-openshift-monitoring-kiali.adoc:23]: I forget, can we link to the [configuring openshift monitoring with service mesh](https://81255--ocpdocs-pr.netlify.app/openshift-service-mesh/latest/metrics/ossm-metrics-assembly#ossm-config-openshift-monitoring-only_ossm-metrics-assembly) section here or is that against the style guide?

Comment (gwynnemonahan) [reply modules/ossm-config-openshift-monitoring-kiali.adoc:23]: That is against style. xrefs are not allowed in modules.

Comment (gwynnemonahan) [reply modules/ossm-config-openshift-monitoring-kiali.adoc]: An "Additional resources" section has been added with a link to "Configuring OpenShift Monitoring with Service Mesh."

**Review by nrfox — APPROVED** [review]

**Comment by pbajjuri20** [comment]
LGTM..!

**Review by pbajjuri20 — APPROVED** [review]

File: kiali/ossm-kiali-assembly.adoc [hunk 2]
```diff
@@ -14,5 +14,11 @@ include::modules/ossm-install-kiali-operator.adoc[leveloffset=1]
 
 include::modules/ossm-config-openshift-monitoring-kiali.adoc[leveloffset=1]
 
+[role="_additional-resources"]
+[id="additional-resources_{context}"]
+== Additional resources
```
Comment (jeana-redhat) [inline kiali/ossm-kiali-assembly.adoc:19]: These workaround "Additional resources" are typically done with the "trailing include" version of the [Additional resources](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) header:
```suggestion
[role="_additional-resources"]
.Additional resources
```

**Review by jeana-redhat — COMMENTED** [review]
One nit, otherwise LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: kiali/ossm-kiali-assembly.adoc [hunk 3]
```diff
@@ -14,5 +14,10 @@ include::modules/ossm-install-kiali-operator.adoc[leveloffset=1]
 
 include::modules/ossm-config-openshift-monitoring-kiali.adoc[leveloffset=1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../metrics/ossm-metrics-assembly.html#ossm-config-openshift-monitoring-only_ossm-metrics-assembly[Configuring OpenShift Monitoring with Service Mesh]
```
Comment (adellape) [inline kiali/ossm-kiali-assembly.adoc:20]:
```suggestion
* xref:../metrics/ossm-metrics-assembly.adoc#ossm-config-openshift-monitoring-only_ossm-metrics-assembly[Configuring OpenShift Monitoring with Service Mesh]
```

**Review by adellape — COMMENTED** [review]
Hold merge for `s/html/adoc/` in `xref`.

**Review by adellape — APPROVED** [review]

**Pull request merged**